packages/flutter/lib/src/cupertino/search_field.dart

```diff
@@ -275,6 +278,18 @@ class CupertinoSearchTextField extends StatefulWidget {
   /// {@macro flutter.widgets.Focus.focusNode}
   final FocusNode? focusNode;
 
+  /// {@macro flutter.material.textfield.onTap}
+  final VoidCallback? onTap;
+
+  /// {@macro flutter.widgets.editableText.autocorrect}
+  final bool autocorrect;
+
+  /// Disables the text field when false.
```

Comment:
nit: make the original docs in `CupertinoTextField` a macro?

Reply:
This was my first though, but I noticed that `CupertinoSearchTextField` does not have a `prefix`/`suffix` property that is being mentioned in the API docs in the original `CupertinoTextField` and this may cause the docs test to fail (similar to https://github.com/flutter/flutter/pull/79966#issuecomment-815255754)

Reply:
I see that the problem is just that the property names are different: `CupertinoTextField` has a `suffix` proeprty however in `CupertinoSearchTextField` it is called `suffixIcon`. Also `CupertinoSearchTextField` does not have a `prefix` property (it is added in the build method) so we can't reference it in the API docs.

Those are the reasons we can't reuse the docs. I will improve the docs to better describe the behavior of `enabled`.